test/Swashbuckle.AspNetCore.SwaggerGen.Test/XmlComments/XmlMethodInfoExtensionsTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Linq;
+using System.Reflection;
+using Swashbuckle.AspNetCore.SwaggerGen.Test.Fixtures;
+using Xunit;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen.Test
+{
+    public class XmlMethodInfoExtensionsTests
+    {
+        [Theory, MemberData(nameof(DifferentMethodsSignatures))]
+        public void DifferentMethodsSignatures_ShouldNotBeNull(MethodInfo methodInfo)
+        {
+            Assert.NotNull(methodInfo.GetUnderlyingGenericTypeMethod());
```

Comment:
Is there anything meaningful to assert on the result other than it doesn't throw and isn't null? Feels like this could return the wrong thing and still pass.

Reply:
Applied a strategy to change the parameterNames of every method and the result types to be different